src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
```

Comment:
Yes sorry, ProgressiveEnhancement is effectively abolished in this release in favour of ContextAwareness. I neglected to delete the docs page referring to it